apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -62,6 +62,8 @@ stop_apps() ->
     ?SLOG(notice, #{msg => "stopping_emqx_apps"}),
     _ = emqx_alarm_handler:unload(),
     ok = emqx_conf_app:unset_config_loaded(),
+    %% Mute otel deps application.
+    _ = emqx_otel:stop_otel(),
```

Comment:
Nit: could we move this call to `emqx_otel_app:stop/1`?

Reply:
stop_otel is call application:stop/1 to stop other application, It's can't use in otel_app:stop or prep_stop.

Reply:
Maybe we should impl a emqx_machine behaiour callback prep_stop ?

Reply:
Good point.  Having an emqx_machine callback sounds like a great idea.  I don't think it's necessary for this PR, though.